wsmaster/pom.xml

```diff
@@ -42,7 +42,6 @@
         <module>che-core-api-ssh-shared</module>
         <module>wsmaster-local</module>
         <module>che-core-sql-schema</module>
-        <module>integration-tests</module>
```

Comment:
Maybe then it makes more sense to move integration tests folder out from wsmaster?

Reply:
For example? What if `assembly/integration-tests`? Does it make sense to do that in this or we can do that in separate?

Reply:
Like `/tests/integrations-tests`?

> Does it make sense to do that in this or we can do that in separate?

I don't mind

Reply:
`tests/integrations-tests` the difficulty that I can see is that `tests/integrations-tests` probably should run before assembly and `test/selenium`  after.